flexget/plugins/clients/qbittorrent.py

```diff
@@ -62,31 +66,31 @@ def connect(self, config):
         'localhost'.
         """
         self.session = Session()
-        self.url = 'http://{}:{}'.format(config['host'], config['port'])
+        self.url = '{}://{}:{}'.format('http' if config['use_ssl']==False else 'https', config['host'], config['port'])
         if config.get('username') and config.get('password'):
```

Comment:
1. You probably meant the opposite, `https` if `use_ssl` is True. 
2. Explicitly checking if a statement is True should be made with the `is` comparator: `if var is True:`
3. It's more python to check for truthiness directly:
`'https' if config['use_ssl'] else 'http'`

Reply:
Forget about 1, misread